src/unused.js

```diff
@@ -0,0 +1 @@
+console.log('Ooops!);
```

Comment:
There is a syntax error: the string is not closed properly with a quote. Make sure to close the string with a matching single or double quote.